experimentation/views/analysis_enrollments.view.lkml

```diff
@@ -1,10 +1,7 @@
 view: analysis_enrollments {
   parameter: slug {
-    type: unquoted
-    suggest_explore: experimenter_experiments
```

Comment:
What's weird about this is that if you do this:

```
suggest_explore: suggest_explore: experimenter_experiments
suggest_dimension: normandy_slug
```

The suggestions (and everything else) seems to work, but the validation fails, saying it can't find the dimension (even though it exists in the experimenter_experiments explore).

The docs do say this: 

> [in addition, ensure that the view is joined to the Explore that is specified in the suggest_explore parameter.](https://cloud.google.com/looker/docs/reference/param-field-suggest-explore)

Thus the full outer join below, even though it doesn't seem to be necessary 🤷